lib/util.mli

```diff
@@ -0,0 +1,49 @@
+(** The Util module contains a number of reusable functions
+    handy for multicore testing. *)
+
+
+val repeat : int -> ('a -> bool) -> 'a -> bool
+(** [repeat num prop] iterates a (non-determistic) property [prop]
```

Comment:
Same here.

Reply:
Good point :+1: 
I agree that the comment could be clearer. I've tried to rephrase it in commit 8e78a8c